src/Command/GithubReleaseCommand.php

```diff
@@ -0,0 +1,162 @@
+<?php declare(strict_types=1);
+
+namespace SymfonyDocsBuilder\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Exception\RuntimeException;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Dotenv\Dotenv;
+use Symfony\Component\HttpClient\Exception\ClientException;
+use Symfony\Component\HttpClient\HttpClient;
+use Symfony\Component\Process\Exception\ProcessFailedException;
+use Symfony\Component\Process\Process;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+class GithubReleaseCommand extends Command
```

Comment:
Having a command to release a command-line library feels weird to me - there's no reason, for example, that this should even been accessible when you're working with the phar file. Could we make this a flat `bin/create_release` file? We would still require the autoloader... and then probably we could refactor some of the code into "service classes" that are used in that script (so that the script is ultimately tiny). We could then *not* include this script nor those supporting classes in the phar file.